doc/manual/oqum/hazard/01a_logic_trees.tex

```diff
@@ -208,7 +208,7 @@ \subsection{Logic trees as described in the nrml schema}
 	contains the name of a ground motion prediction equation (a list of
 	available GMPEs can be obtained using \texttt{oq info -g} or 
 	\texttt{oq info --gsims} and these are also documented at:
-	\href{http://docs.openquake.org/oq-engine/2.8/openquake.hazardlib.gsim.html}{http://docs.openquake.org/oq-engine/2.8/openquake.hazardlib.gsim.html}):
+	\href{http://docs.openquake.org/oq-engine/3.7/openquake.hazardlib.gsim.html}{http://docs.openquake.org/oq-engine/3.7/openquake.hazardlib.gsim.html}):
 
```

Comment:
Use the stable links, for instance https://docs.openquake.org/oq-engine/stable/openquake.hazardlib.gsim.html